staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/accessors.go

```diff
@@ -71,14 +75,15 @@ type WebhookAccessor interface {
 }
 
 // NewMutatingWebhookAccessor creates an accessor for a MutatingWebhook.
-func NewMutatingWebhookAccessor(uid, configurationName string, h *v1.MutatingWebhook) WebhookAccessor {
-	return &mutatingWebhookAccessor{uid: uid, configurationName: configurationName, MutatingWebhook: h}
+func NewMutatingWebhookAccessor(uid, configurationName string, cluster logicalcluster.LogicalCluster, h *v1.MutatingWebhook) WebhookAccessor {
```

Comment:
I think you can undo this and just get the cluster from `h` inside the function (rather than passing in cluster)

Reply:
@ncdc `h` has no ObjectMeta